webapp/channels/src/components/post/post_error_boundary.tsx

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import withErrorBoundary from 'components/with_error_boundary';
+
+export function withPostErrorBoundary<P>(component: React.ComponentType<P>) {
+    return withErrorBoundary<P>(component, {
+        renderFallback: ({clearError}) => {
+            return (
+                <div className='a11y__section post'>
```

Comment:
maybe add too: `aria-live='assertive'`

Reply:
I don't think we want `aria-live` on these since that would cause these to be read out whenever a user visits a channel with an erroring post. I think we only want to use `aria-live` for something really urgent like losing the WS connection or for something that's directly in response to one of their actions like form errors or autocomplete results